bundle.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package tests
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	spec "github.com/opencontainers/specs/specs-go"
+)
+
+const image = "busybox"
+
+const tmpDir = "/tmp"
+
+const configTemplate = "src/github.com/kata-containers/tests/data/config.json"
+
+// Bundle represents the root directory where config.json and rootfs are
+type Bundle struct {
+	// Config represents the config.json
+	Config *spec.Spec
+
+	// Path to the bundle
+	Path string
+}
+
+// NewBundle creates a new bundle
+func NewBundle(workload []string) (*Bundle, error) {
+	path, err := ioutil.TempDir(tmpDir, "bundle")
```

Comment:
You can delete `tmpDir` and pass an empty string as the first param here as it will default to using a sensible temporary directory:

```
ioutil.TempDir("", "bundle")
```